index.html

```diff
@@ -162,22 +162,21 @@ <h2 class="heading-36"> About us </h2>
         <!-- Here add two coln-->
         <div id="about-us" class="flex">
           <div id="about-part-1">
-            <!-- Add Column 1 here / TODO -->
-            <span class="span-des">SOME THINGS ABOUT Us</span>
-            <div class="flex-box-1">
-              <p class="p1 commanp"><span class="special-border">01</span></p>
+            <span class="span-des">SOME THINGS ABOUT US</span>
```

Comment:
Give this an id like `about-us-list`
and then instead of giving each and every list an id, just use
`getElementsByClassName`, this will give a an array of items and then you can easily loop and add the styles.

Just think of it as in future, it becomes so easy to just add a list item that id specifically need not be added.

Reply:
Ok will try that out.